### ISSUE TITLE
VariationalAutoencoderRunner.py (cost= nan)

### ISSUE THREAD

**Issue by daviddao** [tensorflow/models#23]
When running the VAE example `VariationalAutoencoderRunner.py` in models/autoencoder , I get following output:

```
Epoch: 0001 cost= nan
Epoch: 0002 cost= nan
Epoch: 0003 cost= nan
Epoch: 0004 cost= nan
Epoch: 0005 cost= nan
Epoch: 0006 cost= nan
Epoch: 0007 cost= nan
Epoch: 0008 cost= nan
Epoch: 0009 cost= nan
Epoch: 0010 cost= nan
Epoch: 0011 cost= nan
Epoch: 0012 cost= nan
Epoch: 0013 cost= nan
Epoch: 0014 cost= nan
Epoch: 0015 cost= nan
Epoch: 0016 cost= nan
Epoch: 0017 cost= nan
Epoch: 0018 cost= nan
Epoch: 0019 cost= nan
Epoch: 0020 cost= nan
```

Machine: AWS EC2 GPU instance with image ami-77e0da1d

**Comment by martinwicke** [comment]
@snurkabill, and idea?

**Comment by snurkabill** [comment]
VAE is defined for scaled numbers into [0, 1] (http://arxiv.org/pdf/1312.6114v10.pdf), are you sure that you have correctly scaled data? Please, keep me updated.

**Comment by snurkabill** [comment]
Sure I have idea. I don't double checks. VAE needs data scaled between zero and one. Normally scaled data lead to negative logarithm, which is obviously error. Fixed with next PR

**Comment by martinwicke** [comment]
Fixed in #24.
